src/main/java/htmlelements/pages/SearchResultsPage.java

```diff
@@ -0,0 +1,14 @@
+package htmlelements.pages;
+
+import htmlelements.ExtendedMyWebElement;
+import htmlelements.ExtendedWebPage;
+import io.qameta.htmlelements.annotation.Description;
+import io.qameta.htmlelements.annotation.FindBy;
+
+import java.util.List;
+
+public interface SearchResultsPage extends ExtendedWebPage {
+    @Description("Results list")
+    @FindBy("//div[@class='product-tile qa-product-tile __eadf2 col-md-4 col-xs-6 qa-search-result-item']")
```

Comment:
This locator looks like copied from Chrome Dev Tools.
Although it can be helpful to use Chrome Dev Tools generated locators for development or debugging purposes, it is not the best solution for testing framework. 
If you look at the locator, it has multiple random classes like `col-md-4` which is (as far as I remember) dynamically generated class from Angular and it depends on browser viewport. If you run tests on smaller browser, viewport will change, locator will also change and the test will fail.
That is why you need to invent your own, resilient and as much as possible stable locators.
This particular locator can be refactored to:
`.results-list div.product-tile`
Where `.result-list` grabs the node with class `result-list` and then it goes to every `div` node which has `product-tile` class

For CSS selectors you can check: https://saucelabs.com/resources/articles/selenium-tips-css-selectors
For XPath selectors you can check: https://www.lambdatest.com/blog/complete-guide-for-using-xpath-in-selenium-with-examples/